README.md

```diff
@@ -164,7 +164,10 @@ optional arguments:
   -H, --hidden          include hidden (dot) files [default: off]
   -i, --includeimgp     re-process _IMGP files. * RISKY: refer to docs
   -k, --keep            skip (honors -c or --pr) images matching specified
-                        hres or vres or --res=100 [default: off]
+                        hres or vres or --res=100 or
```

Comment:
I can't remember what --res=100 here means. Can you please try to figure it out?